api-reference/_hidden/dxChartCommonAnnotationConfig/image/image.md

```diff
@@ -4,7 +4,7 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-Configures the image to be displayed in the annotation. Applies only if the [type](/api-reference/_hidden/dxChartCommonAnnotationConfig/type.md '/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#type') is *"image"*.
+Configures the image to be displayed in the annotation. Applies only if the [type](/api-reference/_hidden/dxChartCommonAnnotationConfig/type.md '{basewidgetpath}/Configuration/annotations/#type') is *"image"*.
 
 ---
 To display an image, assign its URL to the **url** option. Use the **height** and **width** options to resize the image if needed. Otherwise, you can assign the URL directly to the **image** option.
```

Comment:
```suggestion
To display an image, assign its URL to the **url** option and use the **height** and **width** options to resize the image. Otherwise, assign the URL directly to the **image** option.
```